spec/text-editor-spec.js

```diff
@@ -7314,6 +7314,41 @@ describe('TextEditor', () => {
         editor.toggleLineCommentsForBufferRows(0, 0)
         expect(editor.lineTextForBufferRow(0)).toBe('test')
       })
+
+      it('does not select the new delimiters'), () => {
```

Comment:
Hi. Unfortunately, the closing parentheses after the description string in this test means that the block following the test isn't running at all. So the test is green only because it isn't really running. I'll try and get the test passing, but I may kick this back to you if it ends up being to much work.